lib/config/validation.spec.ts

```diff
@@ -1134,6 +1134,44 @@ describe('config/validation', () => {
       ]);
     });
 
+    it('errors if invalid matchHost values in hostRules', async () => {
+      GlobalConfig.set({ allowedHeaders: ['X-*'] });
+
+      const config = {
+        hostRules: [
```

Comment:
Please add one with a missing matchHost to make sure that _doesn't_ error. e.g. have hostType and token only